README.md for dummies revamp - Blur Bogaloo update (#45)
AryToNeX/Glasscord: README.md

## How do I install it?
Well, glad you asked!

- Look in the Releases section for the latest released version of Glasscord. [Download the `glasscord.asar` file from there](https://github.com/AryToNeX/Glasscord/releases/latest/download/glasscord.asar).
- Locate your Electron app installation folder. We will assume it being the root directory from now on.
- Locate the `resources` folder. Inside it you'll likely have an `app.asar` file **OR** an `app` folder.

> # THIS PART OF THE INSTALLATION DEPEND ON YOUR INSTALLED DISCORD APPLICATION, DON´T FOLLOW BOTH CASE STEPS

Reviewer: ```suggestion
# THIS PART OF THE INSTALLATION DEPEND ON YOUR INSTALLED DISCORD APPLICATION, DON'T FOLLOW BOTH CASE STEPS
```
Consistency: the whole document uses `'` instead of `´`

#### Notes for Discord

> The Glasscord own CSS Loader is currently depreceated in the current state for Discord usage.

Reviewer: Deprecated is not the right word for here, broken would be more adapted. Also this is valid for the latest release of Glasscord but not for the version in this repository. Should the README reflect latest release or latest code?

Reviewer 2: Latest code, always

So you will have to use a third party CSS loader to load Discord themes.

> If you don´t have one installed, please install it FIRST, then install Glasscord AFTER you completed the other installation.

Reviewer: ```suggestion
If you don't have one installed, please install it FIRST, then install Glasscord AFTER you completed the other installation.
```
Consistency: the whole document uses `'` instead of `´`

Doing otherwise MAY BREAK the ability of the third party CSS loader to auto-update itself, so be warned!

The default glasscord theme are adapted for [Powercord](https://github.com/powercord-org/powercord) usage, but [BeautifulDiscord](https://github.com/leovoel/BeautifulDiscord), [EnhancedDiscord](https://github.com/joe27g/EnhancedDiscord) and [BandagedBetterDiscord](https://github.com/rauenzi/BetterDiscordApp) work as well.

> The root discord electron installation folder is habitually situated in :

Reviewer: ```suggestion
The root Discord electron installation folder is habitually situated in :
```

Reviewer 2: I want NO Discord-specific stuff in the README. All Discord specific stuff can be written in the Wiki instead.

>         - Windows: `%appdata%/discord/app-*`; take the latest app version number if you have multiple ones.;

Reviewer: Double punctuation here, and this line is the only one to have a semi.

#### accepts a value between those ones: `acrylic`, `blurbehind`, `transparent`; defaults to `acrylic`
Sets the blur type on Windows.
- `acrylic` refers to the strong blur used in Microsoft's Fluent Design. Note: it can be slow when

> dragging/resizing on some Windows versions, this is a Windows compositor own problem, not a glasscord one.

Reviewer: ```suggestion
dragging/resizing on some Windows versions, this is a Windows compositor own problem, not a Glasscord one.
```

Reviewer 2: dragging/resizing on some Windows versions**. This is a bug on Windows' side, so please don't report it on the issue tracker.**

**Note:** Check the Glasstron project to see which window servers/managers are compatible

## How do i compile it ?

> It is as simple as doing `npm -i` from the glasscord folder.

Reviewer: ```suggestion
It is as simple as doing `npm -i` from the Glasscord folder.
```
